docs/admin/garbage-collection.md

```diff
@@ -42,15 +42,34 @@ to free. Default is 80%.
 We also allow users to customize garbage collection policy through the following kubelet flags:
 
 1. `minimum-container-ttl-duration`, minimum age for a finished container before it is
-garbage collected. Default is 1 minute.
+garbage collected. Default is 0 minute, means any finshed container will be GC.
 2. `maximum-dead-containers-per-container`, maximum number of old instances to retain
-per container. Default is 2.
+per container. Default is 1.
 3. `maximum-dead-containers`, maximum number of old instances of containers to retain globally.
-Default is 100.
+Default is -1, means this flag is disabled.
```

Comment:
For consistency of grammar: "Default is -1, **which** means this flag is disabled."